docs/standard/datetime/working-with-calendars.md

```diff
@@ -134,6 +134,8 @@ Calendars typically divide dates into eras. However, the <xref:System.Globalizat
 > [!IMPORTANT]
 >  A new era in the <xref:System.Globalization.JapaneseCalendar> and <xref:System.Globalization.JapaneseLunisolarCalendar> begins on May 1, 2019. This change affects all applications that use these calendars. See [Handling a new era in the Japanese calendar in .NET](https://blogs.msdn.microsoft.com/dotnet/2018/11/14/handling-a-new-era-in-the-japanese-calendar-in-net/) for more information and to determine whether your applications are affected. See [Prepare your application for the Japanese era change](/windows/uwp/design/globalizing/japanese-era-change) for information on testing your applications on Windows to ensure their readiness for the era change.
 
+An era in most calendars denotes an extremely long time period. In the Gregorian calendar, for example, the current era spans more than two millenia. For the <xref:System.Globalization.JapaneseCalendar> and the <xref:System.Globalization.JapaneseLunisolarCalendar>, the two calendars that support multiple eras, this is not the case. An era corresponds to the period of an emperor's reign. Support for multiple eras, particularly when the upper limit of the current era is unknown, poses special challenges. For more information, see [Handling eras in the Japanese calendars](handling-eras.md).
```

Comment:
I am not sure if there will be any geopol issue here. looks comparing the eras in other calendars with the eras in Japan. would it be better if we just removed the beginning part telling about the eras? and only talk about Japanese eras?

Reply:
There shouldn't be any geopol issue. I think it's important to note that for most calendars, an era is a very long time period, and that most calendars support just one era. The implication is that you don't need to concern yourself with eras unless there's some compelling reason to do so when working with these calendars.